docs/toc.yml

```diff
@@ -1,85 +1,48 @@
-    href: tutorials/excel-tutorial.md
-    displayName: Excel
-  - name: Outlook add-in tutorial
-    href: /outlook/add-ins/addin-tutorial
-    displayName: Outlook
-  - name: PowerPoint add-in tutorial
-    href: tutorials/powerpoint-tutorial.md
-    displayName: PowerPoint
-  - name: Word add-in tutorial
-    href: tutorials/word-tutorial.md
-    displayName: Word
-
-- name: Concepts
-  items:
-  - name: Requirements for running add-ins
-    href: concepts/requirements-for-running-office-add-ins.md
-  - name: Browsers used by Office Add-ins
-    href: concepts/browsers-used-by-office-web-add-ins.md
-  - name: Add-in development best practices
-    href: concepts/add-in-development-best-practices.md
-  - name: Add-in development lifecycle
-    href: concepts/add-in-development-lifecycle.md
-  - name: Privacy and security
-    href: concepts/privacy-and-security.md
-  - name: Performance
-    href: concepts/resource-limits-and-performance-optimization.md
-
-- name: How-to guides
-  items:
+    - name: Office Add-ins platform overview
+      href: overview/office-add-ins.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Host and platform availability
+      href: overview/office-add-in-availability.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Explore Office JS API using Script Lab
+      href: overview/explore-with-script-lab.md
+      displayName: Excel, Word, PowerPoint
+    - name: Requirements for running add-ins
+      href: concepts/requirements-for-running-office-add-ins.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+  - name: Concepts
+    items:
+    - name: Add-in development best practices
+      href: concepts/add-in-development-best-practices.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Add-in development lifecycle
+      href: concepts/add-in-development-lifecycle.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Browsers used by Office Add-ins
+      href: concepts/browsers-used-by-office-web-add-ins.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Office versions and requirement sets
+      href: develop/office-versions-and-requirement-sets.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Install the latest Office version
+      href: develop/install-latest-office-version.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Performance
+      href: concepts/resource-limits-and-performance-optimization.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Privacy and security
+      href: concepts/privacy-and-security.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
   - name: Design the User Interface
```

Comment:
I can see your point here. IIRC, it's "Design the User Interface" because editorial guidance suggested we be consistent in using verbs where feasible in that part of the TOC, to represent the phases of building an app (i.e., Design, Develop, Test and debug, Publish) -- and "Design" alone didn't adequately represent that the docs are about designing the UI (as opposed to designing the architecture of your app itself).  

Despite that guidance though -- I agree that "User Interface" would be better here. I'll make that change, and we can always adjust again down the road if we get additional feedback on this.